src/main/java/com/feed_the_beast/ftbu/cmd/CmdChunks.java

```diff
@@ -249,6 +264,12 @@ public String getCommandName()
         }
 
         @Override
+        public int getRequiredPermissionLevel()
```

Comment:
Commands are for OPs only by default, you don't need to specifically say that its getRequiredPermissionLevel() is 4. Also its 2 for all my commands